<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SandboxButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Песочница</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AboutButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QLabel" name="MainLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lumberjack</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Lapis\Learn\Python</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LectureButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Лекции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>740</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TaskButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Песочница</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
